packages/project-access/src/file/file-search.ts

```diff
@@ -0,0 +1,97 @@
+import type { Editor } from 'mem-fs-editor';
+import { basename, dirname, join, sep } from 'path';
+import { default as find } from 'findit2';
```

Comment:
@ullasholla, we've tried `glob` which was 5x-10x slower than `findit2`. As we use `findit2` since quite some time without issues, I'd like to keep it in order to reduce risks of regression. Not saying that we should never swap the find module, but it should be done as a separate step with deeper analysis, which I consider as done for `findit2` due to long term experience.